xknx/telegram/telegram.py

```diff
@@ -40,12 +50,16 @@ def __init__(
         direction: TelegramDirection = TelegramDirection.OUTGOING,
         payload: APCI | None = None,
         source_address: IndividualAddress = IndividualAddress(0),
+        tpdu_type: TPDUType = TPDUType.T_DATA,
+        prio_system: bool = False,
```

Comment:
there is `system, normal, urgent, low` - should we use an Enum and set `low` as default so all 4 can be used in the future?
```suggestion
        priority: Priority = Priority.LOW,
```
In cemi_frame.py we can set the flag according to this Enum.